x-pack/plugins/ml/server/routes/apidoc_scripts/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "../../../tsconfig.json",
+  "extends": "../../../tsconfig.base.json",
```

Comment:
With `x-pack/plugins/ml/tsconfig.json` being deleted, should this be `"../../../../../tsconfig.base.json"` including the file from `x-pack` itself?

Reply:
argghhh. it's not the fist time when I've imported non-existing tsconfig, surprisingly TS doesn't report the problem